### ISSUE TITLE
Restrictions to assert that a run is complete need a fix ?

### ISSUE THREAD

**Issue by loraine-gueguen** [GenomiqueENS/aozan#17]
The miseq run directory has the following files. 
With Aozan-3.0.2, the "new run" step runs smoothly and was added in `var/newrun.done`, but the "end run" step does not start.
Could this be because the `RunParameters.xml` filename is not exactly the one given in the code below ?https://github.com/GenomiqueENS/aozan/blob/24991ea89edfe7f6b7a2100ef09073832e3d4f9b/src/main/java/fr/ens/biologie/genomique/aozan/aozan3/dataprovider/IlluminaRawRunDataProvider.java#L209 

```
jdoe@slurm:/my/path/230224_M02439_0043_000000000-KT42M$ ls -1
Alignment_1
Basecalling_Netcopy_complete_Read1.txt
Basecalling_Netcopy_complete_Read2.txt
Basecalling_Netcopy_complete_Read3.txt
Basecalling_Netcopy_complete_Read4.txt
Basecalling_Netcopy_complete.txt
CompletedJobInfo.xml
Config
CopyComplete.txt
Data
GenerateFASTQRunStatistics.xml
ImageAnalysis_Netcopy_complete_Read1.txt
ImageAnalysis_Netcopy_complete_Read2.txt
ImageAnalysis_Netcopy_complete_Read3.txt
ImageAnalysis_Netcopy_complete_Read4.txt
ImageAnalysis_Netcopy_complete.txt
InstrumentAnalyticsLogs
InterOp
Logs
QueuedForAnalysis.txt
Recipe
ReportInfo.dat
RTAComplete.txt
RunCheckDetail.txt
RunCompletionStatus.xml
RunInfo.xml
RunParameters.xml
SampleSheet.csv
SampleSheet.csv.test
SoftwareVersionsFile.csv
Thumbnail_Images
```

Note: in the `RunParameters.xml` file, we have `<RunParametersVersion>MiSeq_1_2</RunParametersVersion>` and `<RTAVersion>1.18.54.4</RTAVersion>`

**Comment by loraine-gueguen** [comment]
FYI, in the RunParameters.xml file, we have `<RunParametersVersion>MiSeq_1_2</RunParametersVersion>` and `<RTAVersion>1.18.54.4</RTAVersion>`

**Comment by jourdren** [comment]
Hi @loraine-gueguen, I think that the 6e1b2c1 commit has fixed the issue. It seems that RTA on MiSeq produce CopyComplete.txt like with RTA3 but no RTARead1Complete.txt, RTARead2Complete.txt and RTARead3Complete.txt like with RTA2.

**Comment by loraine-gueguen** [comment]
fixed in aozan-3.1-SNAPSHOT
